### PR TITLE
single_node_performance.py: allow overriding data dir

### PR DESCRIPTION
**Pull request by msmouse**

### CODE REVIEW

**Review by grao1991 — APPROVED** [review]

File: testsuite/single_node_performance.py [hunk 1]
```diff
@@ -163,6 +163,11 @@ class RunGroupConfig:
 
 HIDE_OUTPUT = os.environ.get("HIDE_OUTPUT")
 
+DATA_DIR = os.environ.get("DATA_DIR")
+if not(DATA_DIR):
+    print("Please set env_var DATA_DIR")
```
Comment (igor-aptos) [inline testsuite/single_node_performance.py:168]: do we want to default one, or require it to be set?

**Review by igor-aptos — APPROVED** [review]

File: testsuite/single_node_performance.py (hunk 1 above)
Comment (msmouse) [reply testsuite/single_node_performance.py:168]: For the purpose of pn2 I think it's better to require people to set.. just one less thing to ask each case that doesn't perform well.

Comment (igor-aptos) [reply testsuite/single_node_performance.py:168]: actually, this will break regression test on PRs, you definitely need to have default :)

Comment (msmouse) [reply testsuite/single_node_performance.py:168]: I mean I don't land it on main

Comment (msmouse) [reply testsuite/single_node_performance.py:168]: (if we do we update the CI workflows to put in /tmp)

**Review by sitalkedia — APPROVED** [review]

**Pull request merged**